__fixtures__/fluentBitCases.ts

```diff
@@ -1,12 +1,12 @@
 import fs from 'fs';
-import { COMMANDS, FluentBitSchemaType } from '../src/constants';
+import { COMMANDS, FluentBitSection } from '../src/constants';
 
-type Case = [name: string, content: string, SchemaAST: { config: FluentBitSchemaType[] }];
+type Case = [name: string, content: string, SchemaAST: { config: FluentBitSection[] }];
 
 export const cases: Case[] = [
   [
-    './__fixtures__/basic.conf',
-    fs.readFileSync('./__fixtures__/basic.conf', { encoding: 'utf8' }),
+    './__fixtures__/basic/basic.conf',
+    fs.readFileSync('./__fixtures__/basic/basic.conf', { encoding: 'utf8' }),
```

Comment:
So the difference here is instead of storing the filename we're detaining it's contents. How does this deal with errors on file read, encoding, etc.?

Reply:
this is just a loading platform for the tests. I will always create the right encoding.

Reply:
https://github.com/calyptia/fluent-bit-config-parser/issues/19